tests/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+set(
+    sources
+    0001-multiobj.c
```

Comment:
> is there any way to wildcard this?

Yes, but it's not recommended since may lead to problems:
```
We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.
```
* https://cmake.org/cmake/help/latest/command/file.html